allennlp/common/util.py

```diff
@@ -287,9 +287,14 @@ def import_submodules(package_name: str) -> None:
     # Import at top level
     module = importlib.import_module(package_name)
     path = getattr(module, '__path__', [])
+    path_string = '' if len(path) == 0 else path[0]
 
     # walk_packages only finds immediate children, so need to recurse.
-    for _, name, _ in pkgutil.walk_packages(path):
+    for module_finder, name, _ in pkgutil.walk_packages(path):
+        # Sometimes when you import third-party libraries that are on your path,
+        # `pkgutil.walk_packages` returns those too, so we need to skip them.
+        if path_string and module_finder.path not in path_string:
```

Comment:
what's the reasoning for the `in` instead of `==`?

Reply:
Me doing a cursory look at the strings and not realizing they were equal instead of a substring... =)